packages/v4/src/content/design-guidelines/components/drag-and-drop/drag.md

```diff
@@ -0,0 +1,72 @@
+---
+id: Drag and drop
+section: components
+---
+The **drag and drop** interaction can be used to reposition elements on screen into a layout that benefits the user. This gives the user more flexibility to arrange and/or group items without having to make code changes.
+
+## Usage
+### Simple list
+1. Draggable area: The drag and drop interaction can be triggered with click and hold over any part of the item. The fa-grip icon is used to show that the drag and drop interaction is available.
+
+<img src="./img/Simple-list-drag-1.png" alt="Showing the area in which an item can be dragged."/>
+
+2. Bounding box: Upon click & hold a `$pf-color-blue-400` border will show the draggable area that is available. 
+
+3. onDrag event: The list item being dragged will also use a `$pf-color-blue-400` border to highlight it as the item being dragged and all other list items will switch to a disabled state. The space where the item is being dragged from will remain empty to indicate its original position in the list.
+
+<img src="./img/Simple-list-drag-2-3.png" alt="Borders appear around the item and bounding box shows the area in which an item can be dragged."/>
```

Comment:
Could you add a width to all these images so they're less blown up on the screen/ The width should be the width in the artboard in sketch.

So you'd just add width="123"  to the image tag